src/tpm2_pytss/utils.py

```diff
@@ -71,6 +71,47 @@ def make_credential(
     return (credblob, secret)
 
 
+def credential_to_tools(
+    id_object: TPM2B_ID_OBJECT, encrypted_secret: TPM2B_ENCRYPTED_SECRET
+) -> bytes:
```

Comment:
docstring

Reply:
Add some docstrings. I keep this PR draft on purpose because I would like to clean it a bit up, so the docs might change a bit.